doc/manual/src/release-notes/rl-next.md

```diff
@@ -52,3 +52,12 @@
 
```

Comment:
Relatedly, `dirOf` has no effect on the lazy path roots:

```
nix-repl> lazyPath       
/home/tweagysil/src/nix/

nix-repl> dirOf lazyPath
/home/tweagysil/src/nix/
```

But this does make sense, because the lazy path root essentially acts like a filesystem root, and the directory of a filesystem root is the root itself, which Nix also does with `/`:

```
nix-repl> /.                        
/

nix-repl> dirOf /.
/
```

So I don't think this needs any fixes, but it would be good to write a note about this in the docs.